### PR TITLE
kubernetes: add 1.0.1-1.9.4 docs

### PR DESCRIPTION
**Pull request by pires**
## Description

https://jira.mesosphere.com/projects/KUB/issues/KUB-175

## Urgency

- [ ] Blocker 
- [x] High
- [ ] Medium

## Other

Closes #708

### CODE REVIEW

File: pages/services/kubernetes/1.0.1-1.9.4/release-notes/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## Improvements
+
+* [Kubernetes 1.9.4](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.9.md#changelog-since-v193)
+* [etcd 3.3.2](https://github.com/coreos/etcd/blob/master/CHANGELOG-3.3.md#v332-2018-03-08)
+
+## Bug Fixes
+
+* Bump Kubernetes to v1.9.4 to fix [a couple nasty CVEs](https://github.com/kubernetes/kubernetes/issues/60813).
+* Bump `dcos-commons` to v0.40.5, which fixes a nasty bug where this framework
```
Comment (triclambert) [inline pages/services/kubernetes/1.0.1-1.9.4/release-notes/index.md]: s/nasty//

Comment (pires) [reply pages/services/kubernetes/1.0.1-1.9.4/release-notes/index.md]: on it.

**Comment by pires** [comment]
@ToddGreenstein you've just been appointed as our new docs master! Thank you :)

File: pages/services/kubernetes/1.0.0-1.9.3/index.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 layout: layout.pug
 navigationTitle: Kubernetes 1.0.0-1.9.3
 title: Kubernetes 1.0.0-1.9.3
-menuWeight: 10
+menuWeight: 20
```
Comment (MatApple) [inline pages/services/kubernetes/1.0.0-1.9.3/index.md]: Curious why the bump in menuWeight for the old version? New version is set to `10`

Comment (jimmidyson) [reply pages/services/kubernetes/1.0.0-1.9.3/index.md]: This is to ensure strict ordering of versions in version grid - see https://docs.mesosphere.com/services/beta-kubernetes/ for the same output of the beta releases. This weight for old versions is bumped on import of new version to push the new version to the top of the version stack.

Comment (pires) [reply pages/services/kubernetes/1.0.0-1.9.3/index.md]: What the great @jimmidyson said! Thanks, Jimmi.

Comment (MatApple) [reply pages/services/kubernetes/1.0.0-1.9.3/index.md]: Thanks for the explanation @jimmidyson!

**Pull request merged**